dbms/src/Flash/Pipeline/Schedule/Tasks/Task.h

```diff
@@ -104,7 +104,7 @@ class Task
     // To reduce the overheads of `mem_tracker_holder.get()`
     MemoryTracker * mem_tracker_ptr;
 
-    ExecTaskStatus task_status{ExecTaskStatus::INIT};
```

Comment:
Is this change related to the join spill?

Reply:
Because the join spill/restore of the pipeline model will submit the io task asynchronously for execution.
The changes here allow the io task to be submitted directly to the io thread pool instead of being submitted to the cpu thread pool first and then to the io thread pool